test/platform-routes.test.js

```diff
@@ -98,15 +120,49 @@ describe('Platform Routes HTTP request handler', () => {
       ])
     })
   })
+
+  describe('GET /measurements/daily', () => {
+    it('returns daily total accepted measurement count for the given date range', async () => {
+      await givenDailyStationMetrics(pgPool, '2024-01-10', [
+        { station_id: 'station1', accepted_measurement_count: 1 }
+      ])
+      await givenDailyStationMetrics(pgPool, '2024-01-11', [
+        { station_id: 'station2', accepted_measurement_count: 1 }
+      ])
+      await givenDailyStationMetrics(pgPool, '2024-01-12', [
+        { station_id: 'station2', accepted_measurement_count: 2 },
+        { station_id: 'station3', accepted_measurement_count: 1 }
+      ])
+      await givenDailyStationMetrics(pgPool, '2024-01-13', [
+        { station_id: 'station1', accepted_measurement_count: 1 }
+      ])
+
+      const res = await fetch(
+        new URL(
+          '/measurements/daily?from=2024-01-11&to=2024-01-12',
+          baseUrl
+        ), {
+          redirect: 'manual'
+        }
+      )
+      await assertResponseStatus(res, 200)
+      const metrics = await res.json()
+      assert.deepStrictEqual(metrics, [
+        { day: '2024-01-11', accepted_measurement_count: '1' },
+        { day: '2024-01-12', accepted_measurement_count: '3' }
```

Comment:
Need to use strings here because SUM query results in BIGINT. Do we want to cast to an int, or leave as I have it in this PR now?

Reply:
I am okay with using the string representation. I think we are going to overflow INT32 very soon.